docs/en/api/FS-API.md

```diff
@@ -11,18 +11,21 @@ can either use the [Alluxio Java Client](#Java-Client), or the
 [Hadoop-Compatible Java Client](#Hadoop-Compatible-Java-Client), which
 wraps the Alluxio Java Client to implement the Hadoop API.
 
+Alluxio also provides a [POSIX API]({{ '/en/api/FUSE-API.html' | relativize_url }}) after mounting
+Alluxio as a local FUSE volume.
+
 By setting up an Alluxio Proxy, users can also interact with Alluxio through a REST
-API similar to the Filesystem API. The REST API currently has language bindings for
-Go and Python.
+API similar to the Filesystem API. The REST API is currently used for the Go and Python language
+bindings.
 
-A third option is to interact with Alluxio through its S3 API. Users can interact
+A fourth option is to interact with Alluxio through its S3 API. Users can interact
 using the same S3 clients used for AWS S3 operations. This makes it easy to change
 existing S3 workloads to use Alluxio.
 
 * Table of Contents
 {:toc}
 
-## Java Client
+# Java Client
```

Comment:
I think throughout the docs we try to keep the highest level header within docs as h2 (`##`) I would consider keeping this and making the `Alluxio Java API` header h3

Reply:
Updated, thanks.